src/common/hooks/useLockBodyScroll.ts

```diff
@@ -0,0 +1,11 @@
+import { useLayoutEffect } from "react";
+
+export const useLockBodyScroll = () => {
```

Comment:
do you have a magic trick to avoid the viewport to move when changing the property ?

My trick is to compute the size of the bar and add a padding right :see_no_evil: